Updated README.md instructions for Debian builds

Devariiii opened the pull request:
While trying to install the .deb the program would just open and close instantly. After some trail and error I figured out that it was a dependency issue. 

I have updated the requirements.txt so it will download the correct packages (for Debian 12 at least) and updated the README.md instructions on how to install through said file. I have also added a note in case of .deb failure to try to download the dependencies manually.

limburgher commented:
The requirements.txt file isn't a script, it's for use by pip to install modules locally.  Have you tried the deb builds on OBS?

https://build.opensuse.org/package/show/home:gwync/trelby

Devariiii commented:
For the requirements.txt I didn't fully understand what was being done so I reverted my changes.

For the Debian package located at the OpenSuse repositories, it doesn't open unless also using `sudo apt install python3-setuptools wxpython-tools python3-lxml python3-reportlab`. Without that the program will instantly close as soon as it's opened. I've just added the step of installing for Debian packages before installation to the README.md file.

limburgher commented:
Perfect, thank you!

flossybloom67 reviewed (changes requested):
- **```
> `~~![Screenshot_20241227-113735.png](https://github.com/user-attachments/assets/c39948f4-63c0-46fd-becc-5fe4341e27bd)

~~`
```**

flossybloom67 commented:
3f0288fcad8777955fb41f7b40f8e2504628bc0e

flossybloom67 commented on README.md:
```diff
@@ -23,15 +23,19 @@ The major difference of this fork is conversion to Python 3.  I also have a list
 
 #### Debian and variants
```
3f0288fcad8777955fb41f7b40f8e2504628bc0e
